### PR TITLE
Add `--graph` variants for range queries

### PR DESCRIPTION
**Pull request by saswatamcode**
This PR adds a new `--graph` flag for obsctl range queries. It supports two values, `ascii` and `png`.

Output images based on sample data below.

For `ascii`, it outputs directly to terminal,

Screenshot from 2022-07-18 16-36-00

For `png`, it outputs a separate file named `graph<current_time>.png`,

graph2022-07-18 16:34:34 471142544 +0530 IST m=+23 830573890

### CODE REVIEW

**Comment by squat** [comment]
whoa. that's crazy cool

File: pkg/cmd/cmd.go [hunk 1]
```diff
@@ -105,9 +112,111 @@ func handleResponse(body []byte, contentType string, statusCode int, cmd *cobra.
 
 			return fmt.Errorf(jsonErr)
 		default:
-			return fmt.Errorf("request failed with status code %d, error: %s\n", statusCode, string(body))
```
Comment (matej-g) [inline pkg/cmd/cmd.go:108]: Intended drop of the newline at end? I think also maybe this is accidentally `%s` format verb but probably should be `%d` as the one below.

File: pkg/cmd/cmd.go [hunk 2]
```diff
@@ -105,9 +112,111 @@ func handleResponse(body []byte, contentType string, statusCode int, cmd *cobra.
...
+			return fmt.Errorf("request failed with status code %d, error: %s", statusCode, string(body))
 		}
 	}
 
-	return fmt.Errorf("request failed with status code %d\n", statusCode)
+	return fmt.Errorf("request failed with status code %d", statusCode)
+}
+
+func handleGraph(body []byte, graph, query string, cmd *cobra.Command) error {
+	// TODO(saswatamcode): Update spec so that we can use client/models directly.
```
Comment (matej-g) [inline pkg/cmd/cmd.go:123]: This has been updated in observatorium/api, right?

**Review by matej-g — APPROVED** [review]
Absolutely cool addition :sunglasses: Just few minor things.

**Comment by bwplotka** [comment]
Finally, a nice replacement for our Grafana!

**Review by matej-g — APPROVED** [review]
This was already looking great, thanks for adding more tests! Happy to merge this already :blush:

**Pull request merged**